src/homepage/index.html

```diff
@@ -51,9 +51,8 @@ <h2>Emails at the heart of your business logic</h2>
         <li><a href="#roadmap">Roadmap</a></li>
         <li><a href="#posts">Last Posts</a></li>
         <li><a href="#second">Community</a></li>
-        <li><a href="#third">Contribute</a></li>
-        <li><a href="#use-cases">Use-cases</a></li>
```

Comment:
Why getting rid of those two top links though?

Reply:
To have ore space on the top link.

I find it uggly otherwise.

Both IMO are linked to community.